preposterous-kumquat/preposterous-kumquat
# Add sample data and styling

#### What's this PR do?

Add sample data - I hard coded data in by adding a folder with photos so that we could all see the same thing. 
Add some lame CSS styling.
#### Where should the reviewer start?

in Client folder
#### How should this be manually tested?

The data is hard coded to show ice cream photos on the carousel
the upload works but only for valid photos.  We should try to eventually handle invalid types on either the server side or the front end so we don't run in to that issue.
#### Any background context you want to provide?
#### What are the relevant tickets?

45, 3, 5, 6, 7
#### Screenshots (if appropriate)
#### Questions:
